include/boost/decimal/detail/emulated256.hpp

```diff
@@ -89,6 +89,8 @@ struct uint256_t
 
     friend constexpr uint256_t operator*(const uint256_t& lhs, const uint256_t& rhs) noexcept;
 
+    friend constexpr uint256_t operator*(const uint256_t& lhs, const std::uint64_t rhs) noexcept;
```

Comment:
Do we also need `friend constexpr uint256_t operator*(const std::uint64_t lhs, const uint256_t& lhs) noexcept` for completeness?

Reply:
> Do we also need `friend constexpr uint256_t operator* ...`

We need a bunch of them. And I could do a lot more optimizing for mixed-limb left/right. I would, however, prefer to get something into develop and subsequently optimize. Then I canwork on pressing down all the $128$-bit tolerances and other special functions, while optimizing when time arises.